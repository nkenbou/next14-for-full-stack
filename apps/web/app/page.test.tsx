import { composeStories } from "@storybook/react";
import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import * as stories from "./page.stories";

const { Default } = composeStories(stories);

describe("page", () => {
  test("レンダリングすること", () => {
    render(<Default />);
    expect(screen.getByText("web")).toBeInTheDocument();
  });
});
