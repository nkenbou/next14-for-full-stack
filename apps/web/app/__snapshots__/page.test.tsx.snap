// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page レンダリングすること 1`] = `
<code
  class="code"
>
  web
</code>
`;
